spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
     while x != 'boom':
         if welcome():
             password = set_up()
-            scrambled_message = new_message(codex,count)
+            scrambled_message = new_message(codex, count)
             count = 1
         else:
-            retrieve_message(password, scrambled_message,codex,count)
+            retrieve_message(password, scrambled_message, codex, count)
             scrambled_message = None
             count = 0
         x = input("To self-destruct type 'boom'. Otherwise, hit enter. ")
-    self_destruct(5,'KABOOM', "Yes")
+    self_destruct(5, 'KABOOM', "Yes")
+
 
 '''---------------------SUPPORTING FUNCTIONS ------------------------'''
 
 '''Welcomes the user and directs them to either write a new message or retrieve an existing message.'''
+
+
 def welcome():
     today = date.today()
     print('Welcome Agent. The current date is:', today)
-    desire = input("Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
+    desire = input(
+        "Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
     while desire != 'L' and desire != 'R':
```

Comment:
`while desire not in ['L', 'R']` is an alternative approach, though I'm not sure it's more readable